WooCommerce/Classes/ViewRelated/Dashboard/Stats v4/StoreStatsDataOrRedactedView.swift

```diff
@@ -70,6 +72,8 @@ private extension StoreStatsDataOrRedactedView {
         switch state {
         case .redacted, .redactedDueToJetpack:
             redactedView.showJetpackImage = state == .redactedDueToJetpack
+        case .redactedDueToCustomRange:
+            redactedView.showInfoIcon = state == .redactedDueToCustomRange
```

Comment:
We should not need to compare the state here:

```suggestion
            redactedView.showInfoIcon = true
```